selfdrive/controls/controlsd.py

```diff
@@ -746,7 +746,6 @@ def publish_logs(self, CS, start_time, CC, lac_log):
       controlsState.alertText2 = current_alert.alert_text_2
       controlsState.alertSize = current_alert.alert_size
       controlsState.alertStatus = current_alert.alert_status
-      controlsState.alertBlinkingRate = current_alert.alert_rate
```

Comment:
the `if current_alert` above this shouldn't be there. it should be collapsed into a single codepath with an "empty alert. @deanlee want to fix up?